ch.threema.threema-web-desktop.yml

```diff
@@ -42,7 +42,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/threema-ch/threema-web-electron.git
-        commit: d0f631164c8aba34a317fec6ee1ad56de515d0ce
+        commit: 55a1cb27b39a4a0f092be9586868c9abd59f76c2
```

Comment:
Could be replaced by `tag: 1.2.41`, right?

Reply:
`1.2.46` at this point. I've tested it (via direct RPM on Fedora), and it connects fine.